exercise-3.sql

```diff
@@ -0,0 +1,107 @@
+-- 1) Write a SQL query to find the total salary of employees who is in Tokyo excluding whose first name is Nancy
+select sum(salary) from (select e.first_name,e.salary,e.department_id,d.location_id from employees as e inner join departments as d on e.department_id = d.department_id inner join locations as l on d.location_id = l.location_id where l.city = 'Seattle' and lower(e.first_name) != 'nancy');
+
+-- 2) Fetch all details of employees who has salary more than the avg salary by each department.
+select * from employees e inner join departments d on e.department_id = d.department_id where e.salary > (select avg(salary) from employees where department_id = e.department_id);
+
```

Comment:
You are using aggregate function and no group by (in the  sub-query ) ?
Does it fetch the right result ?